### PR TITLE
bump(main/socat): 1.8.0

### PR DESCRIPTION
**Pull request by basilgello**
Signed-off-by: Vasyl Gello <vasek.gello@gmail.com>

### CODE REVIEW

File: packages/socat/build.sh [hunk 1]
```diff
@@ -2,11 +2,12 @@ TERMUX_PKG_HOMEPAGE=http://www.dest-unreach.org/socat/
 TERMUX_PKG_DESCRIPTION="Relay for bidirectional data transfer between two independent data channels"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.7.4.4
+TERMUX_PKG_VERSION=1.8.0.0
 TERMUX_PKG_SRCURL=http://www.dest-unreach.org/socat/download/socat-$TERMUX_PKG_VERSION.tar.gz
```
[the remark this answers is not in the document]
Comment (Grimler91) [reply packages/socat/build.sh:6]: There is no configure script in that archive though, so we will also need to:

```
termux_step_pre_configure() {
    autoreconf -vfi
}
```

[the remark this answers is not in the document]
Comment (basilgello) [reply packages/socat/build.sh]: Fixed (along with new SHA256 sum) & force-pushed!

[the remark this answers is not in the document]
Comment (Grimler91) [reply packages/socat/build.sh]: Seems to be a quite big cleanup needed to make autoreconf work, feel free to drop the url update for now and I'll merge this!

[the remark this answers is not in the document]
Comment (basilgello) [reply packages/socat/build.sh:6]: OK, pushed tne working variant. Lets ping Dr. Fischer about SSL certificate - his main company website is slso on HTTP ehile private area is on LE HTTPS :)

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**